demo/vue-viewer/src/components/Thumbnails.vue

```diff
@@ -61,6 +62,12 @@ export default {
       this.thumbClicked(index, false);
     },
   },
+  computed: {
+    ...mapState(['inputFileName']),
+    canShowThumbnails() {
+      return this.inputFileName.split('.').pop() === 'pdf';
```

Comment:
If we upload an image ?? Will not be shown ??

🤔

Reply:
solved in 67b7256